### DISCUSSION TITLE
ssl connection issue

### DISCUSSION THREAD

**Discussion by techtide-web** [vercel/community#3823] [Help]
### Summary

i created my vercel website with namecheap domain . all the dns configuration are done but it is showing insecure connection . what to do ?

### Steps to Reproduce

can you please give me the steps to solve this error

**Comment by swarnava** [comment]
Hi, What is your domain name?

**Comment by rpatel15-hue** [comment]
had a user with similar issue today, not sure how to debug, most users are fine, one of the individuals was having issues.
